Zilon.Core/CDT.LAST.MonoGameClient/Screens/ScoresScreen.cs

```diff
@@ -1,11 +1,131 @@
-﻿using Microsoft.Xna.Framework;
-
-namespace CDT.LAST.MonoGameClient.Screens
+﻿namespace CDT.LAST.MonoGameClient.Screens
 {
+    using System;
+
+    using Engine;
+
+    using Resources;
+
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Xna.Framework;
+    using Microsoft.Xna.Framework.Graphics;
+    using Microsoft.Xna.Framework.Input;
+
+    using Zilon.Core.Scoring;
+
+    /// <summary>
+    /// Scores screen to show a user's score when a character died.
+    /// </summary>
     internal class ScoresScreen : GameSceneBase
     {
-        public ScoresScreen(Game game) : base(game)
+        private readonly GlobeSelectionScreen _globeGenerationScene;
+
+        private readonly TextButton _goToMainMenu;
+
+        private readonly TextButton _goToNextScreen;
+
+        private readonly TextButton _restartButton;
+
+        private readonly string _scoreSummary;
+
+        private readonly SpriteBatch _spriteBatch;
+
+        /// <inheritdoc />
+        public ScoresScreen(Game game, SpriteBatch spriteBatch)
+            : base(game)
+        {
+            _spriteBatch = spriteBatch;
+
+            _globeGenerationScene = new GlobeSelectionScreen(game: game, spriteBatch: spriteBatch);
+
+            var buttonTexture = game.Content.Load<Texture2D>("Sprites/ui/button");
+            var font = Game.Content.Load<SpriteFont>("Fonts/Main");
+
+            _restartButton = new TextButton(
+                title: UiResources.StartGameButtonTitle,
+                texture: buttonTexture,
+                font: font,
+                rect: new Rectangle(x: 150, y: 150, width: 100, height: 20));
+            _restartButton.OnClick += RestartButtonClickHandler;
+
+            _goToMainMenu = new TextButton(
+                title: UiResources.MainMenuButtonTitle,
+                texture: buttonTexture,
+                font: font,
+                rect: new Rectangle(x: 350, y: 150, width: 100, height: 20));
+            _goToMainMenu.OnClick += GoToMainMenuButtonClickHandler;
+
+            _goToNextScreen = new TextButton(
+                title: UiResources.NextScreenButtonTitle,
+                texture: buttonTexture,
+                font: font,
+                rect: new Rectangle(x: 550, y: 150, width: 100, height: 20));
```

Comment:
Do not use magic numbers. Instead use named constants.

Reply:
What are magic numbers? It's just a position of the button and other parameters.